armbian/base/customize-armbian-rockpro64.sh

```diff
@@ -571,7 +571,7 @@ PartOf=bitcoind.service
 [Service]
 EnvironmentFile=/etc/electrs/electrs.conf
 EnvironmentFile=/mnt/ssd/bitcoin/.bitcoin/.cookie.env
-ExecStartPre=/opt/shift/scripts/systemd-electrs-startpre.sh
+ExecStartPre=+/opt/shift/scripts/systemd-electrs-startpre.sh
```

Comment:
What does the plus here do?

Reply:
It runs the `systemd-electrs-startpre.sh` as root, despite the service user set to `electrs`. This is necessary to recreate the `.cookie.env` file and set permissions:

https://github.com/digitalbitbox/bitbox-base/blob/6373a39f7007271f9ccb0911ab341355cf589552/armbian/base/scripts/systemd-electrs-startpre.sh#L19-L21

Reply:
Thanks for the explanation